src/commands/mobile_app/upload.rs

```diff
@@ -27,7 +30,7 @@ pub fn make_command(command: Command) -> Command {
 pub fn execute(matches: &ArgMatches) -> Result<()> {
     let path_strings = matches.get_many::<String>("paths").unwrap();
 
-    let mut paths: Vec<&Path> = Vec::new();
+    let mut normalized_zips: Vec<TempFile> = Vec::new();
     for path_string in path_strings {
```

Comment:
Type can be inferred here

```suggestion
    let mut normalized_zips = Vec::new();
```